docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+| ----------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Remote codec](#remote-codec) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+The remote codec refers to exposing a Payload Codec through HTTP endpoints for remote encoding and decoding.
+It can be configured using a remote codec server, where each developer can control the mapping locally through the hosts file.
+However, distributing encryption keys for decoding payloads poses a security risk.
+The authorization header can be set on requests to the codec server.
+
+## Replayer
+
+| Feature               | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Replayer](#replayer) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+A Replay is the method by which a Workflow Execution resumes making progress.
+During a Replay the Commands that are generated are checked against an existing Event History.
+Replays are necessary and often happen to give the effect that Workflow Executions are resumable, reliable, and durable.
+
+For more information, see [Deterministic constraints](#deterministic-constraints).
+
+If a failure occurs, the Workflow Execution picks up where the last recorded event occurred in the Event History.
+
+## Retries
+
+| Feature             | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Retries](#retries) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Retry Policy is a collection of attributes that instructs the Temporal Server how to retry a failure of a Workflow Execution or an Activity Task Execution.
+
+## SAGA
+
+| Feature       | Go | Java | TypeScript | Python | .Net | PHP |
+| ------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [SAGA](#saga) | No | Yes  | No         | No     | No   | Yes |
+
+The saga pattern is a design approach for distributed systems where a task extends across machine or microservice boundaries.
+For these tasks, it's crucial to ensure the complete execution of all steps, as partial execution can lead to undesirable outcomes.
+
+Compensating actions, often referred to as compensating transactions, simulate the atomic execution of operations that are distributed across multiple databases in distributed systems.
+If any of these distributed operations fail, its effects are reversed using a compensating action.
+These compensating actions are integral to the broader saga pattern.
+
+Temporal provides a dedicated Saga library, making it easier to handle these distributed tasks.
+You can register functions as compensations, and Temporal manages their execution:
+[Java SDK Documentation](https://www.javadoc.io/static/io.temporal/temporal-sdk/1.0.0/io/temporal/workflow/Saga.html)
+
+For those using other SDKs, you can implement functions like `addCompensation` and `compensate` to achieve similar behavior.
+
+## Sandbox
```

Comment:
I think this is very runtime specific and probably not worth a high-level discussion across languages